R/MixedModelsCommon.R

```diff
@@ -1765,6 +1844,8 @@ gettextf <- function(fmt, ..., domain = NULL)  {
         estimate =  emmContrast[i, ncol(emmContrast) - 4],
         se       =  emmContrast[i, "SE"],
         df       =  emmContrast[i, "df"],
+        lowerCI  =  emmContrast[i, ncol(emmContrast) - 4] + stats::qt(0.025, df = emmContrast[i, "df"]) * emmContrast[i, "SE"],
```

Comment:
should we use instead of hard-coded `0.025` and `0.975` use user defined values here?